src/pages/components/WebView.qml

```diff
@@ -189,21 +189,20 @@ WebContainer {
                             if (size) {
                                 sizes.push(size)
                             }
-                        } else {
-                            sizes = data.href.match(/\d+/)
```

Comment:
Don't read anything from the url. This was big no no as the number might have been from sha1 (cdn url).